etp-front/src/pages/valvonta-kaytto/manager.svelte

```diff
@@ -151,8 +151,12 @@
     id={valvonta.id}
     {toimenpide}
     {templatesByType}
-    {henkilot}
-    {yritykset}
+    henkilot={Toimenpiteet.isOrder(toimenpide)
+      ? henkilot
+      : R.filter(Osapuolet.isOmistaja, henkilot)}
+    yritykset={Toimenpiteet.isOrder(toimenpide)
+      ? yritykset
+      : R.filter(Osapuolet.isOmistaja, yritykset)}
```

Comment:
Me filteröidään osapuolet lista komponentin ulkopuolella, ts. tieto siitä mitä komponentti tekee näillä vuotaa komponentin ulkopuolelle.

Voitaisiin joko:
1. Filteröidä komponentin sisällä niinkuin ennenkin
1. Muokata komponenttien jotka ovat kiinnostuneet vain omistajista parametrien nimeämistä ja kommentoida tähän että filteröidään näin koska se vastaa reaalimaailmaa, ts. Ara ei ole kiinnostunut alun jälkeen muista kuin omistajista

Reply:
Katso onko nyt hyvä